policyengine_us/parameters/gov/irs/ald/student_loan_interest/reduction/start.yaml

```diff
@@ -0,0 +1,66 @@
+description: IRS allows for a student loan interest deduction of this amount, based on filing status.
+
+metadata:
+  unit: currency-USD
+  period: year
+  label: Student loan interest deduction amount
+  breakdown:
+    - filing_status
+  reference:
+    - title: 26 U.S. Code § 221 - Interest on education loans (b)(1)
+      href: https://www.law.cornell.edu/uscode/text/26/221
+    - title: 2021 IRS Publication 970 Tax Benefits for Education
+      href: https://www.irs.gov/pub/irs-prior/p970--2021.pdf#page=37
+    - title: 2022 IRS Publication 970 Tax Benefits for Education
+      href: https://www.irs.gov/pub/irs-prior/p970--2022.pdf#page=36
+    - title: 2023 IRS Publication 970 Tax Benefits for Education
```

Comment:
Cite the annual IRS documents (pointed in #4593) for consistency, and to include 2024 values.